rust/connlib/shared/src/device_id.rs

```diff
@@ -1,7 +1,7 @@
-use tokio::fs;
+use std::fs;
```

Comment:
Gave up on `tokio::fs` for this since the Linux CLI client doesn't have a Tokio runtime. I personally like async FS operations but it's not a big deal, especially since it's only used at startup.